src/components/actions/AddNewProductBtn.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div id="product-control">
+    <div class="product-control__icon" @click="createNewProductHandler"></div>
```

Comment:
is this component used in more than one place? 🤔  
I'd say that it's a bit too small, simple and specialised component to extract it.
Also, this should probably be a button to increase accessibility. 😃